saskatoon/harvest/signals.py

```diff
@@ -37,6 +38,30 @@ def changed_by(sender, instance, **kwargs):
     else:
         instance.changed_by = None
 
+def notify_unselected_pickers(sender, instance, **kwargs):
+    if instance.id:
+        original_instance = sender.objects.get(id=instance.id)
+        if instance.status == "Ready" and original_instance.status != "Ready":
+            unselected_pickers = instance.get_unselected_pickers()
+            email_list = list()
+            mail_subject = _("[Saskatoon] Not Selected as a picker for upcoming Harvest")
+            for p in unselected_pickers:
```

Comment:
Looks like all unselected pickers will be notified each time the pick leaders edits the harvest ?
Maybe we could check and send the emails only when the harvest it passing from the status "Date scheduled" to "Ready". At this time, all required pickers should have been accepted.